api/room/schema.py

```diff
@@ -110,6 +110,8 @@ class Arguments:
     def mutate(self, info, **kwargs):
         validate_empty_fields(**kwargs)
         verify_location_id(kwargs)
+        room_l = kwargs.get('room_labels')
```

Comment:
@Arusey Please could you enlighten on the purpose of `room_l`

Reply:
@daniellamarr PR is still `wip`

Reply:
Yeah, I realized that later. But I didn't want to remove the comment so he sees it before he pushes again.